tests/distributed/synced_batchnorm/unit_test.sh

```diff
@@ -1,8 +1,8 @@
-python python_single_gpu_unit_test.py
-python single_gpu_unit_test.py
-python test_batchnorm1d.py
-python -m torch.distributed.launch --nproc_per_node=2 two_gpu_unit_test.py
-python -m torch.distributed.launch --nproc_per_node=2 two_gpu_unit_test.py --fp16
-python -m torch.distributed.launch --nproc_per_node=2 two_gpu_test_different_batch_size.py --apex
+python python_single_gpu_unit_test.py || exit 1
+python single_gpu_unit_test.py || exit 1
+python test_batchnorm1d.py || exit 1
+python -m torch.distributed.launch --nproc_per_node=2 two_gpu_unit_test.py || exit 1
+python -m torch.distributed.launch --nproc_per_node=2 two_gpu_unit_test.py --fp16 || exit 1
+python -m torch.distributed.launch --nproc_per_node=2 two_gpu_test_different_batch_size.py --apex || exit 1
```

Comment:
Why do we need || exit 1?  Did the processes get stuck?

Reply:
Our CI job doesn't even run this script (yet), but I was finding when I ran it locally that the script wouldn't report a non-zero exit code even if one or more of the tests within this script were failing.  Alternatively, I might be able to add `set -ex` at the top of the script to get similar behavior.